common/utils/src/main/scala/org/apache/spark/internal/LogKey.scala

```diff
@@ -112,6 +113,7 @@ object LogKey extends Enumeration {
   val URI = Value
   val USER_ID = Value
   val USER_NAME = Value
+  val VERSION = Value
```

Comment:
should it be PYTHON_VERSION, SPARK_VERSION, HIVE_VERSION, JAVA_VERSION etc. ?

Reply:
Thanks for the suggestion! Using `PYTHON_VERSION` sounds good to me this case for clarity.